advocacy_docs/partner_docs/ImpervaDataSecurityFabric/05-UsingImpervaDataSecurityFabric.mdx

```diff
@@ -3,61 +3,63 @@ title: 'Using'
+
 Some of these options are: 
-- Scan and assess the servers 
-- Configure security policies 
-- Create and/or manage reports
-- Configure and view audited traffic
-- Configure or work with a Database Cluster   
 
-For in depth information on the many pieces of Database Activity monitoring with the Imperva Data Security Fabric Agent, please visit the Database Activity Monitoring User Guide on the Imperva Website: [Database Activity Monitoring](https://docs.imperva.com/bundle/v14.9-database-activity-monitoring-user-guide/page/70414.htm)
+- Scan and assess the servers.
+- Configure security policies.
+- Create and manage reports.
+- Configure and view audited traffic.
+- Configure or work with a database cluster.  
+
+For in-depth information on the many pieces of database activity monitoring with the Imperva Data Security Fabric agent, see [Database Activity Monitoring User Guide](https://docs.imperva.com/bundle/v14.9-database-activity-monitoring-user-guide/page/70414.htm) on the Imperva website. 
 
-## Monitor EDB Postgres Advanced Server or PostgreSQL Database Traffic
+## Monitor EDB Postgres Advanced Server or PostgreSQL database traffic
 
-One of the uses of the Imperva Data Security Fabric Agent is monitoring database traffic for compliance and auditing. Many regulations and industry standards require organizations to monitor and track data access and changes for auditing purposes. This can include tracking access to sensitive data such as financial information or personal data of customers.
+One of the uses of the Imperva Data Security Fabric agent is monitoring database traffic for compliance and auditing. Many regulations and industry standards require organizations to monitor and track data access and changes for auditing purposes. This can include tracking access to sensitive data, such as financial information or customers' personal data.
 
-Imperva Agents within the Imperva Data Security Fabric solution can be used to track and review the actions of employees and 3rd parties to ensure they are complying with policies and procedures.
+You can use Imperva agents within the Imperva Data Security Fabric solution to track and review the actions of employees and third parties to ensure they're complying with policies and procedures.
 
-## Set-up to Monitor Database Traffic with Imperva Data Security Fabric Agent
+### Setting up to monitor database traffic with Imperva Data Security Fabric agent
 
-The following steps need to be taken in order to set-up the Imperva Data Security Fabric Agent to monitor EDB Postgres Advanced Server traffic.
+Set up the Imperva Data Security Fabric agent to monitor EDB Postgres Advanced Server traffic.
 
-1. Install Imperva Data Security Fabric Agent.
+1. Install Imperva Data Security Fabric.
```

Comment:
@ebgitelman Is it clear what you mean if you omit the word "agent"? To me it is not.